tests/unit/tracing/test_agent_trace.py

```diff
@@ -10,7 +11,7 @@ def create_llm_span(input_tokens: int = 100, output_tokens: int = 50) -> AgentSp
         status=Status(),
         context=SpanContext(span_id=123),
         attributes={
-            "gen_ai.operation.name": "call_llm",
+            OPERATION: "call_llm",
             "gen_ai.usage.input_tokens": input_tokens,
             "gen_ai.usage.output_tokens": output_tokens,
```

Comment:
Why not have usage as a constant enum as well?

Reply:
Missed these (and input/output cost). Pushed an update